Form/Field/ObjectChoiceListItem.php

```diff
@@ -26,9 +27,11 @@ public function __construct(array &$object, ContentType $contentType){
 			$this->label = '<i class="'.(null !== $contentType->getIcon()?$contentType->getIcon():'fa fa-question').'" data-ouuid="'.$this->value.'"></i>&nbsp;&nbsp;';
 			if(null !== $contentType->getLabelField() && isset($object['_source'][$contentType->getLabelField()])){
 				$this->label .= $object['_source'][$contentType->getLabelField()];
+				$this->title = $object['_source'][$contentType->getLabelField()];
```

Comment:
I think this title attribute still has all the html markup (e.g <i class=".." oouid=".."></i>Realtitle)